src/hc-06.cpp

```diff
@@ -0,0 +1,65 @@
+// ==========================================================================
```

Comment:
Incorrect use of Doxygen, please read the wiki for the correct Doxygen standard.